.github/workflows/chatops-binder.yaml

```diff
@@ -0,0 +1,34 @@
+#./github/workflows/chatops-binder.yaml
```

Comment:
did you mean to name this file `Chatops` and use that name in this file?

Reply:
Fixed :)